CHANGELOG.md

```diff
@@ -1,5 +1,76 @@
 # Changelog
 
+## 8.0.0
+
+Teleport 8.0 is a major release of Teleport that contains new features, improvements, and bug fixes.
+
+### New Features
+
+#### Windows Desktop Access 
+
+#### TLS Routing
+
+#### AWS CLI
+
+Teleport adds a new command `tsh aws` to obtain AWS credentials using Teleport SSO/RBAC.
+
+#### Application and Database Automatic Discovery
+
+With auto discovery users can ask Teleport to discover databases and applications in their infrastructure and automatically connect them instead of manually setting them up.
+
+#### WebAuthn
+
+Webauthn support enables Teleport users to use modern second factor devices - apple watch, touch ID.
```

Comment:
Does Apple Watch actually support WebAuthn? In what way?

Would it be better to say:

`WebAuthn support enables Teleport users to use modern second factor options, including Apple FaceID and TouchID.`